frontend/public/i18n.js

```diff
@@ -108,10 +108,9 @@ export const init = () => {
       },
       saveMissing: true,
       missingKeyHandler: function (lng, ns, key) {
-        const error = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}".`;
-        window.windowError = [...(window.windowError || []), error]; // OCPBUGS-45139: store all i18n errors for easy debugging
+        window.windowError = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}".`;
```

Comment:
I think instead of doing a full revert we can keep OCPBUGS-44920 fixed by appending new errors to the windowError string